src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -242,7 +174,7 @@
             <span>Undo</span>
           </v-btn>
           <v-btn
-            v-else-if="isBcCompany"
+            v-else-if="isBcCompany || isBcUlcCompany || isBenefitCompany"
```

Comment:
What about CCC?

Reply:
(changeFrom) CCC not included on UX.

Reply:
Please confirm with UX designer.

Reply:
We've confirmed with UX designer and BA and John Lane

Here's the matrix currently:

![Uploading image.png…]()

Reply:
https://preview.uxpin.com/c21a28a1679da5e2b71044251e8296995e00d9c0#/pages/162402442/simulate/sitemap

Reply:
Straight from Dana:

![image](https://github.com/bcgov/business-edit-ui/assets/3484109/69c4d519-4941-4466-b28f-8bfadfbef367)

Reply:
Perfect, thanks.